openshift-install-powervs

```diff
@@ -61,6 +62,7 @@ EOF
 }
 
 RELEASE_VER=${RELEASE_VER:-"4.15"}
```

Comment:
Might as well bump to 4.20

Reply:
Updated the script to point to 9.6 version for RHCOS and 4.20 version for OCP.